Multi input combox boxes

Quafadas opened the pull request:
Adds multi input combo bboxes. 

Basically a pattern match of combo boxes..

Quafadas commented:
Not 100% sure about the collection type on `HasItems`... but it seems to do sometrhing

sherpal commented on web-components/src/main/scala/be/doeraene/webcomponents/ui5/MultiComboBox.scala:
```diff
@@ -0,0 +1,68 @@
+  object events {
+    val onChange: EventProp[EventWithPreciseTarget[Ref]] = new EventProp("change")
+    val onInput: EventProp[EventWithPreciseTarget[Ref]]  = new EventProp("input")
+    val onSelectionChange: EventProp[dom.Event with HasDetail[HasItems[MultiComboBoxItem.Ref]]] = new EventProp(
```
I've been using the `&` syntax instead of `with`.

Quafadas replied:
As requested, replaced "with " with &. I also did this in combobox (which is where I shamelessly copy pasted from !)

sherpal commented on web-components/src/main/scala/be/doeraene/webcomponents/ui5/MultiComboBoxItem.scala:
```diff
@@ -0,0 +1,48 @@
+  val selected: ReactiveHtmlAttr[Boolean]           = customHtmlAttr("selected", BooleanAsAttrPresenceCodec)
+  val text: ReactiveHtmlAttr[String]           = customHtmlAttr("text", StringAsIsCodec)
+  val additionalText: ReactiveHtmlAttr[String] = customHtmlAttr("additional-text", StringAsIsCodec)
+
```
Even if there are none, I usually put the `slots` and `events` object. It shows that it's not been forgotten but that they are genuinely non-existent.

Quafadas replied:
Done... thnaks.

sherpal reviewed (changes requested):
Just two tiny details to (try to) have full consistency among the codebase :)

Quafadas commented:
HI, Sorry for the delay. I've made the changes as you requested. 

I also updated Combobox to use &... and I seemed to need a "default" slot (which I also added) in order to add items to it.

sherpal commented:
Default slot is not need actually. They all have a default slot. Not using the slot is equivalent to use the "default" slot.
But it doesn't hurt!

Thanks for this! We will discover if the publish functionality works.
